src/test/java/net/sf/jabref/exporter/layout/format/HTMLCharsTest.java

```diff
@@ -21,14 +21,14 @@ public void testBasicFormat() {
 
         Assert.assertEquals("h&aacute;llo", layout.format("h\\'allo"));
 
-        Assert.assertEquals("&#305; &#305;", layout.format("\\i \\i"));
-        Assert.assertEquals("&#305;", layout.format("\\i"));
-        Assert.assertEquals("&#305;", layout.format("\\{i}"));
-        Assert.assertEquals("&#305;&#305;", layout.format("\\i\\i"));
+        Assert.assertEquals("&imath; &imath;", layout.format("\\i \\i"));
```

Comment:
Ah, sorry, wrong answer. :-) The answer above holds for HTMLConverter...

I think HTMLChars format to HTML? Now, since there are named entities these are used, if not, the numerical are used.